features/step_definitions/environments.yml

```diff
@@ -24,3 +24,12 @@ staging:
     Stub Idp Demo One: https://stub-idp-staging.cloudapps.digital/stub-idp-demo-one/
     Stub Idp Demo Two: https://stub-idp-staging.cloudapps.digital/stub-idp-demo-two/
     Stub Idp Disconnected: https://stub-idp-staging.cloudapps.digital/stub-idp-disconnected/
+
+integration:
+  frontend: https://www.integration.signin.service.gov.uk
+  test-rp: https://test-rp-integration.cloudapps.digital/test-rp
+  idps:
+    Stub Country: https://ida-stub-idp-integration.cloudapps.digital/eidas/stub-country/
```

Comment:
Shouldn't all of these be `stub-idp-integration`?

Reply:
Yes they should if we are going to use the `cloudapps.digital` domain but when we originally moved stub-idp it was decided that we would keep the `idp-stub-integration.ida.digital.cabinet-office.gov.uk` hostname to avoid confusing RPs who were used to it from when stub-idp was still in UKCloud. There is already a cdn-route service with that hostname (bound to the old app) so it won't be hard to move that to the new stub-idp.

The question is should we continue to use the `idp-stub-integration.ida.digital.cabinet-office.gov.uk` hostname. If the answer is yes, then our acceptance tests should use that too. If the answer is no then what @CharlesIC says - `stub-idp-integration.cloudapps.digital` is correct.